tools/serve/serve.py

```diff
@@ -378,15 +378,17 @@ def build_routes(aliases):
 
 
 class ServerProc(object):
-    def __init__(self):
+    def __init__(self, name=None):
```

Comment:
If any other review wonders, yes there is another case `wrapper = ServerProc()` where `name` isn't given, and that seems fine.